studio.assetmanager.ams.yaml

```diff
@@ -0,0 +1,77 @@
+app-id: studio.assetmanager.ams
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: ams
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # Access to network
+  - --share=network
+  # Access to X11
+  - --socket=x11
+  # Access to wayland
+  - --socket=wayland
```

Comment:
it's either, or, not both.

Reply:
I copied this from Signal's https://github.com/flathub/org.signal.Signal/blob/0e46425c4bafd2076351bd6d1244a11d38fc1e2d/org.signal.Signal.yaml#L12-L15

Reply:
two wrongs don't make a right

Reply:
Biggest issue, is that when I see things like that it makes it appear as if that is the supported way for wayland support since it is on an actual published flatpak.     Electron can use x11 AND wayland, so my assumption is it would need access to both to choose which method to use.